### PR TITLE
Fix examples (#33)

### PR DESCRIPTION
**Pull request by henrij22**

### CODE REVIEW

File: examples/kirchhoffplate.cc [hunk 1]
```diff
@@ -51,28 +51,24 @@ int main(int argc, char **argv) {
   double lambdaLoad = 1 * std::pow(thk, 3);
 
   const int subsample = postProcessParameters.get<int>("subsample");
-  std::cout << "ALUGRID_VERBOSITY_LEVEL1: " << getenv("ALUGRID_VERBOSITY_LEVEL") << std::endl;
   // Log the Parameters
   spdlog::info(
-      "Filename: {} \n The following parameters were used: \nMaterial: E {}, nu {} \nRefinements: global {}, u {}, v "
+      "Filename: {} \nThe following parameters were used: \nMaterial: E {}, nu {} \nRefinements: global {}, u {}, v "
```
Comment (rath3t) [inline examples/kirchhoffplate.cc:56]: Maybe we can throw out the dependency on spdlog replace this with std::format

File: examples/linear2dsolid.cc [hunk 2]
```diff
@@ -20,10 +18,10 @@
 #include "dune/iga/nurbsgrid.hh"
```
Comment (rath3t) [inline examples/linear2dsolid.cc:18]: can you change the double quotes to angle brackets

**Review by rath3t — CHANGES_REQUESTED** [review]
except for removing `spdlog`

File: examples/kirchhoffplate.cc (hunk 1 above)
Comment (henrij22) [reply examples/kirchhoffplate.cc:56]: Doesn't ikarus use spdlog in the  `init.hh`?

File: examples/linear2dsolid.cc (hunk 2 above)
Comment (henrij22) [reply examples/linear2dsolid.cc:18]: Done

File: examples/kirchhoffplate.cc (hunk 1 above)
Comment (rath3t) [reply examples/kirchhoffplate.cc:56]: Ah yes, this should then be changed there also. But can you still remove it here and create an Issue at Ikarus?

Comment (henrij22) [reply examples/kirchhoffplate.cc:56]: Alright!

**Pull request merged**